indexer/packages/kafka/src/config.ts

```diff
@@ -23,6 +23,8 @@ export const kafkaConfigSchema = {
   KAFKA_REBALANCE_TIMEOUT_MS: parseInteger({ default: 120_000 }),
   KAFKA_HEARTBEAT_INTERVAL_MS: parseInteger({ default: 5_000 }),
   KAFKA_CONCURRENT_PARTITIONS: parseInteger({ default: 1 }),
+  // The number of messages to process before committing the offset.
+  AUTO_COMMIT_THRESHOLD: parseInteger({ default: 100 }),
```

Comment:
suggestion: prefix all kafka variables with `KAFKA_` for legibility. Also consider `KAFKA_CONSUMER_` for the consumer specific ones